Nos/Views/CompactNoteView.swift

```diff
@@ -159,6 +159,9 @@ struct CompactNoteView: View {
         .task {
             await note.loadViewData()
         }
+        .task {
+            await note.loadAttributedContent()
```

Comment:
This is already done in` loadViewData()`, is it needed?

Reply:
Yep! `loadViewData()` has a `guard` set up so that it only really does work once. However if we don't have the author name downloaded then we need to do the work of computing the attributed string again once we get it. That's what this is for.